NEWS.md

```diff
@@ -1,6 +1,7 @@
 # mlr3extralearners 0.4.7
```

Comment:
Please bump version here and in description to 0.4.8

Reply:
done!